MaterialDesignThemes.Wpf/PopupBox.cs

```diff
@@ -767,7 +836,7 @@ protected override void OnMouseLeftButtonUp(MouseButtonEventArgs e)
 
         private void ToggleButtonOnPreviewMouseLeftButtonUp(object sender, MouseButtonEventArgs mouseButtonEventArgs)
         {
-            if (PopupMode == PopupBoxPopupMode.Click || !IsPopupOpen) return;
+            if ((PopupMode == PopupBoxPopupMode.Click | PopupMode == PopupBoxPopupMode.ClickOrFocusWithin) || !IsPopupOpen) return;
```

Comment:
Why use the unary | here? I think || is the better fit.

Reply:
Fixed.
Thanks.